src/js/components/Paragraph/doc.js

```diff
@@ -29,3 +29,21 @@ export const doc = Paragraph => {
 
   return DocumentedParagraph;
 };
+
+export const themeDoc = {
+  'global.colors.text': {
+    description: 'The text color used inside the Paragraph.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: "{ dark: '#f8f8f8', light: '#444444' }",
+  },
```

Comment:
We also need to include `global.edgeSize.<size>` and `global.edgeSize.responsiveBreakpoint`, which are used in `genericStyles()` within StyledParagraph.

Reply:
Done. 
Not 100% sure about the global.edgeSize.<size>, and on whether it should document all sizes. Let me know what you think.